src/data/default.js

```diff
@@ -5,6 +5,7 @@ const data = require('../../static/locales/' + config.locale + '.json');
 data.title = config.title;
 data.header_title = config.headerTitle;
 data.locale = config.locale;
+data.dateTimeLocale = config.dateTimeLocale;
```

Comment:
Snake case please like the other mustache keys.
```suggestion
data.date_time_locale = config.dateTimeLocale;
```